draft-ietf-idr-deprecate-as-set-confed-set.xml

```diff
@@ -526,7 +533,7 @@ R4 - 192.0.2.192/26 AS_PATH "64504 64501"
 
       <section title="Achieving Consistent Origin-AS During Aggregation">
         <t>In the three scenarios above, the aggregating AS 64500 is using
-        traditional brief style aggregation.  This results in inconsistent
+        traditional brief aggregation.  This results in inconsistent

```

Comment:
brief-style?

Reply:
You were varyingly using "brief aggregation", "brief-style aggregation" and "brief style aggregation" but all meant the same. I tried to retain only one form "brief aggregation". Eliminated the attachment of 'style'.